parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -445,15 +445,6 @@ write_files:
   content: !!binary |
     {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
 
-{{- if IsKubeletConfigFileEnabled}}
```

Comment:
when there are no more top level if statements in cloud init, I can stop...

Reply:
basically need to be able to have this file be totally empty or everything come from vhd